### PR TITLE
ADR OWASP ZAP

### PR DESCRIPTION
**Pull request by jorg3lopez**
# ADR for DAST Scanning - OWASP ZAP

The team decided on using OWASP ZAP for DAST scanning. 

## Changes
- Added a new 011-dast.md to the ADR directory.

## Issue
#77

### CODE REVIEW

**Review by sfradkin — COMMENTED** [review]
Did we look into any other DAST tools for comparison?

File: adr/011-dast.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# 11. OWASP ZAP Dynamic Application Security Testing
+
+Date: 2023-01-06
```
Comment (halprin) [inline adr/011-dast.md:3]:
```suggestion
Date: 2023-02-06
```

**Review by halprin — COMMENTED** [review]
Looks good, but I don't think this ADR was done a month ago.  ;)

**Comment by jorg3lopez** [comment]
> Did we look into any other DAST tools for comparison?

I looked into a few other. What made me decide for OWASP ZAP was that OWASP ZAP had a more extensive scan, it had a docker image, can be ran using a GUI, and it could be integrated as part of the workflow without adding dependencies to our code. Other tools that were better came with a subscription, OWASP ZAP is open source and is one of the most preferred  DAST tools. Here is the list of other open source scanning tools that I took a look at:

List
- Wapiti
- w3af
- sqlmap
- Grabber

**Review by halprin — APPROVED** [review]
Boom!

**Pull request merged**